File: level1/나누어 떨어지는 숫자 배열/Solution.java
import java.util.ArrayList;
import java.util.Arrays;

class Solution {
    public int[] solution(int[] arr, int divisor) {
        ArrayList<Integer> al = new ArrayList<>();
        for (int i : arr) {
            if (i % divisor == 0) {
                al.add(i);
            }
        }
        if (al.size() == 0) {
            al.add(-1);
        }
        int[] answer = al.stream().mapToInt(Integer::intValue).toArray();
        Arrays.sort(answer);
        return answer;
    }
}

File: level1/나누어 떨어지는 숫자 배열/Solution.py
def solution(arr, divisor):
    answer = list()
    for i in arr :
        if(i%divisor == 0) : answer.append(i)
    if len(answer) < 1 : return [-1]
    return sorted(answer)
  
  # [i for i in range(0,5)] == [0,1,2,3,4] 
  # for문을 바로 배열에 담을 수도 있구나
